tests/test_validate_metadata.py

```diff
@@ -0,0 +1,57 @@
+import sys
+import unittest
+
+sys.path.append('../validation')
+from validate_metadata import *
+
+
+class TestValidateMetadata(unittest.TestCase):
+    def setup_validation(self, args):
+        args_list = args.split(' ')
+        args = create_parser().parse_args(args_list)
+        filetsv = args.input_metadata
+        metadata = Cell_Metadata(filetsv)
+        metadata.validate_format()
+        metadata.file.close()
+        return metadata
+
+    def test_format_name(self):
+        """Header row of metadata file should start with NAME keyword
```

Comment:
This is a great unit test!  Let's return to it in our upcoming discussion on testing strategy.